### ISSUE TITLE
SIGTRAP Exception Applite crashes

### ISSUE THREAD

**Issue by vsenn** [milanvarady/Applite#67]
```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Applite [1922]
Path:                  /Applications/Applite.app/Contents/MacOS/Applite
Identifier:            dev.aerolite.Applite
Version:               1.2.4 (10)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2024-05-16 16:38:06.1174 +0300
OS Version:            macOS 14.4.1 (23E224)
Report Version:        12
Anonymous UUID:        A5E76A61-97F2-C45C-FAA1-A0E34B9D4040


Time Awake Since Boot: 2400 seconds

System Integrity Protection: enabled

Crashed Thread:        1

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000001, 0x00000001045bd5fc

Termination Reason:    Namespace SIGNAL, Code 5 Trace/BPT trap: 5
Terminating Process:   exc handler [1922]

```

```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Applite [3126]
Path:                  /Applications/Applite.app/Contents/MacOS/Applite
Identifier:            dev.aerolite.Applite
Version:               1.2.4 (10)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2024-05-16 16:56:27.2873 +0300
OS Version:            macOS 14.4.1 (23E224)
Report Version:        12
Anonymous UUID:        A5E76A61-97F2-C45C-FAA1-A0E34B9D4040


Time Awake Since Boot: 3500 seconds

System Integrity Protection: enabled

Crashed Thread:        7

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000001, 0x00000001001495fc

Termination Reason:    Namespace SIGNAL, Code 5 Trace/BPT trap: 5
Terminating Process:   exc handler [3126]
```
etc...

**Comment by tcmonster** [comment]
same issue

**Comment by milanvarady** [comment]
Thanks for reporting the problem!

The crash is fixed in the new [v1.2.5](https://github.com/milanvarady/Applite/releases/tag/v1.2.5) update. The crash will most likely be resolved once [this pull request](https://github.com/Homebrew/homebrew-cask/pull/174018) is merged, until then you can manually install the new version to fix the problem.

**Comment by vsenn** [comment]
Thank you for a quick fix!!
It works after re-install
```
brew uninstall --cask applite
brew install --cask applite

```

Shows v1.24 (10) though 😄 
![image](https://github.com/milanvarady/Applite/assets/25433230/9c7cd22d-24f3-41d8-aac0-38dcbcc07dc1)
